metadata-ingestion/src/datahub/ingestion/source/dremio/dremio_aspects.py

```diff
@@ -366,6 +366,9 @@ def _create_browse_paths_containers(
     ) -> Optional[BrowsePathsV2Class]:
         paths = []
 
+        if self.platform_instance:

```

Comment:
This should now automatically be taken care of by this -> https://github.com/datahub-project/datahub/pull/12045